Library/Formula/gl2ps.rb

```diff
@@ -14,8 +14,10 @@ class Gl2ps < Formula
 
   depends_on "cmake" => :build
   depends_on "libpng"
+  depends_on :x11 => :optional
 
   def install
+    inreplace "CMakeLists.txt", "find_package(GLUT)", "" if build.without? "x11"
```

Comment:
Yes, I think we could request an explicit `ENABLE_GLUT` option to be added, hopefully with default `OFF` but probably fine if they want default `ON`.